### PR TITLE
Load override name

### PR DESCRIPTION
**Pull request by PSRCode**
Introduce --override-name for the load command

### CODE REVIEW

File: src/lib/lttng-ctl/load.c [hunk 1]
```diff
@@ -106,10 +108,12 @@ const char *lttng_load_session_attr_get_override_ctrl_url(
 {
 	const char *ret = NULL;
 
-	if (attr && attr->override_attr->ctrl_url) {
-		ret = attr->raw_override_ctrl_url;
+	if (!attr || !attr->override_attr) {
+		goto end;
 	}
 
+	ret = attr->raw_override_ctrl_url;
```
Comment (jgalar) [inline src/lib/lttng-ctl/load.c]: Shouldn't we still check for `ctrl_url` before returning the "raw" value?

File: src/lib/lttng-ctl/load.c [hunk 2]
```diff
@@ -118,10 +122,12 @@ const char *lttng_load_session_attr_get_override_data_url(
 {
 	const char *ret = NULL;
 
-	if (attr && attr->override_attr->data_url) {
-		ret = attr->raw_override_data_url;
+	if (!attr || !attr->override_attr) {
+		goto end;
 	}
 
+	ret = attr->raw_override_data_url;
```
Comment (jgalar) [inline src/lib/lttng-ctl/load.c]: Shouldn't we still check for `data_url` before returning the "raw" value?

File: src/lib/lttng-ctl/load.c [hunk 3]
```diff
@@ -546,6 +589,53 @@ int lttng_load_session_attr_set_override_url(
...
+	}
+end:
+	return ret;
+}
+
+/*
+ * The lttng-ctl API does not expose all the information needed to load the
+ * session configurations. Thus, we must send a load command to the session
+ * daemon which will, in turn, load its current session configuration.
+ */
```
Comment (jgalar) [inline src/lib/lttng-ctl/load.c]: I'll remove this, seems like a rebase artifact.

Comment (PSRCode) [inline src/lib/lttng-ctl/load.c]: +1

**Comment by PSRCode** [comment]
Fixed the comments stuff and check before using raw data

**Comment by jgalar** [comment]
Merged, thanks!

**Pull request closed without merging**